mobile/src/components/app/app.tsx

```diff
@@ -5,15 +5,20 @@ import RNBootSplash from 'react-native-bootsplash';
 import 'fast-text-encoding';
 import { Root as RootNavigation } from '~/navigation/root/root.navigation';
 import { styles } from './styles';
+import { ENV } from '~/common/enums/enums';
 
 const App: FC = () => {
   const handleNavigationReady = (): void => {
     RNBootSplash.hide({ fade: true });
   };
 
+  const linking = {
```

Comment:
Move this to src/config/linking.config.ts

Reply:
good idea, thx